src/main/java/bookup/Book.java

```diff
@@ -14,11 +14,11 @@ public Book(String isbn) {
 		this.isbn = isbn;
 	}
 
-	public String getisbn() {
+	public String getISBN() {
```

Comment:
So NOW we're doing capital?!

Reply:
If anything should at least be `getIsbn()`. I would be ok with that.